sendlib.c

```diff
@@ -959,24 +966,7 @@ static size_t convert_file_from_to(FILE *fp, const char *fromcodes, const char *
   int ncodes, i, cn;
 
   /* Count the tocodes */
-  ncodes = 0;
-  for (c = tocodes; c; c = c1 ? c1 + 1 : 0)
-  {
-    c1 = strchr(c, ':');
-    if (c1 == c)
-      continue;
-    ncodes++;
-  }
-
-  /* Copy them */
-  tcode = mutt_mem_malloc(ncodes * sizeof(char *));
```

Comment:
Mh, we should probably have not merged this yet. This did segfault when you try to write a mail. At least that is what I was looking into yesterday. At line 986 we will try to write to `tcode` but that is now always `NULL`.

Reply:
I agree - that should be reverted.

Reply:
sorry, reverted